packages/backend/src/server/api/endpoints/notes/create.ts

```diff
@@ -78,11 +78,11 @@ export const meta = {
 		},
 
 		fileIds: {
-			validator: $.optional.arr($.type(ID)).unique().range(1, 4),
+			validator: $.optional.arr($.type(ID)).unique(),
 		},
 
 		mediaIds: {
-			validator: $.optional.arr($.type(ID)).unique().range(1, 4),
+			validator: $.optional.arr($.type(ID)).unique(),
```

Comment:
Maybe we should validate that the array has at least one element if it is present, but from what I can see it should also work without that validation, since it will be replaced by an empty array if it is not present: https://github.com/outloudvi/misskey/blob/32ec42a89f0ae6d40e047f84e1bef2c0f6820551/packages/backend/src/server/api/endpoints/notes/create.ts#L185-L194